semgrep/semgrep/core_runner.py

```diff
@@ -406,7 +405,7 @@ def get_files_for_language(
             ) from ex
         return list(targets)
 
-    def _get_targets(
+    def _create_core_run_planner(
```

Comment:
```suggestion
    def _plan_core_run(
```

the "create planner" name implies that this function returns a callable that can plan the run, but the return value is actually the plan itself